sdk/keyvault/azure-security-keyvault-keys/src/main/java/com/azure/security/keyvault/keys/cryptography/SignatureHashResolver.java

```diff
@@ -16,14 +16,14 @@ class SignatureHashResolver {
         DEFAULT.put(SignatureAlgorithm.ES256, HashAlgorithm.SHA_256);
         DEFAULT.put(SignatureAlgorithm.ES256K, HashAlgorithm.SHA_256);
         DEFAULT.put(SignatureAlgorithm.ES384, HashAlgorithm.SHA_384);
-
         DEFAULT.put(SignatureAlgorithm.ES512, HashAlgorithm.SHA_512);
         DEFAULT.put(SignatureAlgorithm.RS256, HashAlgorithm.SHA_256);
         DEFAULT.put(SignatureAlgorithm.RS384, HashAlgorithm.SHA_384);
         DEFAULT.put(SignatureAlgorithm.RS512, HashAlgorithm.SHA_512);
         DEFAULT.put(SignatureAlgorithm.PS256, HashAlgorithm.SHA_256);
         DEFAULT.put(SignatureAlgorithm.PS512, HashAlgorithm.SHA_512);
         DEFAULT.put(SignatureAlgorithm.PS384, HashAlgorithm.SHA_384);
+        DEFAULT.put(SignatureAlgorithm.EDDSA, HashAlgorithm.SHA_512);
```

Comment:
This seems correct, but we should verify with @daviddesberg, Vick, or anyone from the MHSM team.

Reply:
Seems this needs to be SHA256. See email conversation for more details.

Reply:
Will change this to `SHA_256` as that is the only algorithm currently supported by the service.